scoring.js

```diff
@@ -11,33 +11,50 @@ const calculateScore = () => {
                 player.stats.rushing.fumbles * -3
             )
         case 'RB':
-            break
+            return (
+                player.stats.rushing.yards / 10 +
+                player.stats.rushing.touchdowns * 6 +
+                player.stats.rushing.fumbles * -3 +
+                player.stats.receiving.receptions * 1 +
+                player.stats.receiving.yards / 10 +
+                player.stats.receiving.touchdowns * 6 +
+                player.stats.receiving.fumbles * -3 +
+                player.stats.return.kickreturn.yards / 15 +
+                player.stats.return.kickreturn.touchdowns * 6 +
+                player.stats.return.kickreturn.fumbles * -3 +
```

Comment:
You can make separate functions to calculate passing, rushing, receiving and returns and interchange them between players that use them.